kafka/src/main/java/com/expedia/adaptivealerting/kafka/KafkaConfigProps.java

```diff
@@ -28,9 +28,15 @@
     // App names
     // ================================================================================
     
+    /**
+     * Anomaly Detector Mapper KStreams app name.
+     */
+    public static final String ANOMALY_DETECTOR_MAPPER = "anomaly-detector-mapper";
+    
     /**
      * Metric Router KStreams app name.
      */
+    @Deprecated
```

Comment:
Nice to add some docs for a deprecated vale.

Reply:
@djsutho Done, thanks sir.